packages/dcos-history/extra/history/server.py

```diff
@@ -90,4 +92,10 @@ def start():
 
     state_buffer = BufferCollection(os.environ['HISTORY_BUFFER_DIR'])
     BufferUpdater(state_buffer, headers_cb).run()
-    app.run(host='0.0.0.0', port=int(os.environ.get('PORT', '15055')))
+
+
+def start():
```

Comment:
Having a testing shortcut is fine in general, but the UX of this shortcut is somewhat awkward. Specifically, I need to be aware of which directory I call it from and have files be in the right place. Before this change, you can call dcos-history wherever and it still works. Alternatively, this could pass the config path and app module location via arguments or change the working directory to always be in the right place, but my personal (and lazy) preference would be a README line on how to run it and be done